src/processor/Package.ts

```diff
@@ -60,3 +75,17 @@ export class Package {
     }
   }
 }
+
+function isDuplicateDefinition(
+  array: NamedExportable[],
+  resource: NamedExportable,
+  type: string
+): boolean {
+  if (array.find(e => e.name === resource.name)) {
```

Comment:
I'm wondering if we want to deduplicate based on name, since SUSHI can produce two profiles with the same `name`, for example, the following:
```
Profile: MyPatient
Parent: Patient
* ^name = "foo"

Profile: MyPatient2
Parent: Patient
* ^name = "foo"
```
is FSH that runs without error, but then causes a duplicate error when I run the output of SUSHI through GoFSH. In SUSHI I think we use the combination of `resourceType` and `id` to detect duplicates. Could we do something similar here?

Reply:
Ugh.  So that probably is the best way to do it.  But IIRC there were some complications for implementation in trying to keep both definitions and number them.  I think that was the first approach @jafeltra was experimenting with -- but I'll let her comment on it.

Reply:
OK, just documented where we landed on this:

- first, deduplicate the incoming JSON by `resourceType` and `id` -- logging an error and tossing the duplicate
- then, on the export side, detect duplicate `name`s and log an error for them -- but still let the duplicates through anyway.  The FSH will be invalid, but these we'll leave it to the author to decide how to resolve that.
- there shouldn't be many duplicate names in the previous step if we've already removed all the duplicates by `resourceType` and `id`

Reply:
I believe that 1dc1f5e covers these changes.